hack/update-payload-crds.sh

```diff
@@ -4,7 +4,6 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 #TODO(jerzhang): once MOSC/MOSB graduates, update the v1 crds to include them
 crd_globs="\
-    authorization/v1/zz_generated.crd-manifests/*_config-operator_*.crd*yaml\
```

Comment:
This likely isn't a good idea because it prevents future CRDs added here from being added to the payload, but I'm doing it this way temporarily for testing as the `RoleBindingRestriction` CRD is the only one that exists in this directory for now.

Reply:
After fiddling with bash globbing and trying to figure out how to match everything but the rolebindingrestrictions crd yaml file for a while it seems like it might be easier to just remove this and any new CRDs added to this path get an explicit glob entry that would ignore the rolebindingrestrictions CRD file.